manage/deploying/production/heroku_do.rst

```diff
@@ -0,0 +1,226 @@
+using our email and email password to sign up.
+
+After having your account ready, we will need to create a droplet.
+A droplet is the way Digital Ocean calls its own Virtual Private Server (VPS). 
+Click on the Create Droplet button on the right corner of the screen.
+
+.. image:: ../images/create_droplet.png
+    :align: center
+    :alt: Create droplet
+
+We will be redicrect to the droplet selection page. 
+In here, we need to check the OS for your Droplet. In this tutorial, we choose the Ubuntu 16.04 OS for our VPS.
+
+.. image:: ../images/create_dropletos.png
+    :align: center
+    :alt: Choose droplet OS
+
+After that, we need to choose the plan for your Droplet. 
+Usually, we will use the $10/month since it provides good space for caching.
+In addition, this plan also prevents some errors that we might get into while deploying our Plone sites.
+
+.. image:: ../images/create_dropletsize.png
+    :align: center
+    :alt: Choose droplet size
+
+.. note::
+    In case you want to deploy your app using a specific domain name, you can learn how to set it up at `Digital Ocean documentation <https://www.digitalocean.com/community/tutorials/how-to-set-up-a-host-name-with-digitalocean>`_
+
+Sign in to your Droplet 
+^^^^^^^^^
+
+Mac and Linux
+`````````
+
+If you are using Mac or Linux environment, you will have to turn on the terminal and type in:
+
+.. code-block:: shell
+
+    ssh root@[your-droplet-ip-adress]
+
+It will ask you to fill in your password. When you created a droplet, an email should be sent to you. 
+It contains the password for your droplet. 
+Use this password to log in to your droplet through the terminal and change your password.
+
+When you are done with it, make sure that you are in the /root directory. Install the Unified Installer 
+
+.. code-block:: shell
+
+    wget --no-check-certificate https://launchpad.net/plone/5.0/5.0.7/+download/Plone-5.0.7-UnifiedInstaller.tgz
+
+Windows
+`````````
+
+On Windows, you will have to download `Putty <http://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html>`_.
+Run the setup. When you are done, start Putty.
+Use your droplet ip address, port 22 and hit enter. It should open a linux terminal for you to fill in your username and password.
+Log in to your droplet with the password from the email that you received.
+
+.. image:: ../images/winputty.png
+    :align: center
+    :alt: Windows putty setup
+
+When you are done, it shold ask you to change the password.
+
+Deploy Plone to VPS
+^^^^^^^^^
+
+When you are in, make sure that you are at the root directory and download the Unified Installer
+
+.. code-block:: shell
+
+    wget --no-check-certificate https://launchpad.net/plone/5.0/5.0.7/+download/Plone-5.0.7-UnifiedInstaller.tgz
+    $ tar -xf Plone-5.0.7-UnifiedInstaller.tgz
+
+Run the the Unified Installer
+
+.. code-block:: shell
+
+    cd Plone-5.0.7-UnifiedInstaller
+    $ ./install.sh $OPTION
+
```

Comment:
This is creating 'double' content, better use :ref: or :doc: for linking to the installation documentation